app/controllers/application_controller.rb

```diff
@@ -50,6 +50,9 @@ def check_privacy_policy_accepted
     policy = PrivacyPolicy.current
     return if policy.nil?
 
+    return if request.format == "application/json"
+    return if controller_name == "cookies"
```

Comment:
That feels a bit dirty.

Can we have a protected `requires_privacy_policy` method on this controller, have it return `true` by default and then in cookies controller we override that method to return `false`? 

And in this place we `return if requires_privacy_policy`?

Reply:
good shout 👍

Reply:
found an even nicer solution!